.eslintrc

```diff
@@ -21,7 +21,8 @@
     "prettier/prettier": [
       "error",
       { "trailingComma": "all", "singleQuote": true }
-    ]
+    ],
+    "jsx-a11y/media-has-caption": "off"
```

Comment:
@jackahl isn't that a useful check in general?

Reply:
Yeah, I would leave that out and add it to a per project config. So we leave it as cannon for Volto.

Reply:
@sneridagh excluding a single line from eslint does not work. No idea why. I had to re-add this line to make Travis pass. I'd say let's cut a release and then re-visit this later. A linter problem should not hold back a release we need.